.github/FAQ.md

```diff
@@ -0,0 +1,105 @@
+# Frequently Asked Questions
+
+## What is the Deno Standard Library?
+
+The Deno Standard Library is a library of packages that aim to provide a robust,
+secure, and modern foundation for building JavaScript and TypeScript
+applications.
+
+## Which APIs does the Standard Library support?
+
+The Standard Library aims to compliment the
+[JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference),
+[Web](https://developer.mozilla.org/en-US/docs/Web/API) and
+[Deno runtime](https://deno.land/api) APIs.
+
+## Which runtimes are compatible with the Standard Library?
+
+By virtue of the APIs it supports, the Standard Library primarily aims to work
+with:
+
+1. [Deno](https://deno.com/)
+1. [Deno Deploy](https://deno.com/deploy)
+1. Web browsers (i.e. [Google Chrome](https://www.google.com.au/chrome/),
+   [Mozilla Firefox](https://www.mozilla.org/firefox/),
+   [Apple Safari](https://www.apple.com/safari/), etc.)
+
+Other runtimes are supported to varying degrees, depending on how much they
+support the same APIs that the Standard Library targets. These runtimes include:
+
+1. [Node.js](https://nodejs.org/)
+1. [Cloudflare Workers](https://workers.cloudflare.com/)
+1. [Bun](https://bun.sh/)
+
+## How is the Standard Library distributed?
+
+The Standard Library is distributed as a library of packages on
+[JSR](https://jsr.io/), a modern package registry for JavaScript and TypeScript.
+
+To learn more about JSR, see the [documentation](https://jsr.io/docs).
+
+## Which version of a given package should I use?
+
+We recommend using the latest version of a given package.
+
+Thanks to JSR, you can constrain dependency versions by defining the SemVer
+range in the import specifier.
+
+E.g. If the latest major version of a package is 1, it should be imported as
+follows:
+
+```ts, ignore
+import { bar } from "jsr:@std/foo@1/bar";
```

Comment:
`deno add <pkg>` adds import map entry with the form of `"@std/foo": "jsr:@std/foo@^1.2.3"`. The above example seems slightly inconsistent with it to me

Reply:
Sorry, what are you suggesting? I changed it to `jsr:@std/foo@^1`. How's that?